nodes/vr_sonic/vr-sonic.go

```diff
@@ -0,0 +1,112 @@
+// Copyright 2020 Nokia
+// Licensed under the BSD 3-Clause License.
+// SPDX-License-Identifier: BSD-3-Clause
+// The author of this code is Adam Kulagowski (adam.kulagowski@codilime.com), CodiLime (codilime.com).
+
+package vr_sonic
+
+import (
+	"context"
+	"fmt"
+	"path"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/srl-labs/containerlab/clab/exec"
+	"github.com/srl-labs/containerlab/nodes"
+	"github.com/srl-labs/containerlab/types"
+	"github.com/srl-labs/containerlab/utils"
+)
+
+var (
+	kindnames          = []string{"vr-sonic"}
+	defaultCredentials = nodes.NewCredentials("admin", "YourPaSsWoRd")
```

Comment:
we tend to use a common username/password combination
it is either `admin:admin` or (if security rules prevent it) `clab:clab@123`

Reply:
Corrected to be `admin:admin`